app/infrastructure/external/email/providers/outlook_provider.py

```
@@ -19,10 +20,20 @@
 class OutlookProvider:
     """Outlook/Office 365 provider using Microsoft Graph API."""
 
-    def __init__(self) -> None:
+    def __init__(self, *, http_client: httpx.AsyncClient | None = None) -> None:
         self._access_token: str | None = None
         self._config: EmailProviderConfig | None = None
         self._graph_url = "https://graph.microsoft.com/v1.0"
+        self._shared_http = http_client
+
+    @asynccontextmanager
+    async def _http_cm(self):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `_http_cm`

(ANN202)